coil-gif/api/android/coil-gif.api

```diff
@@ -48,11 +52,14 @@ public final class coil3/gif/ImageRequestsKt {
 	public static final fun getAnimationStartCallback (Lcoil3/Extras$Key$Companion;)Lcoil3/Extras$Key;
 	public static final fun getAnimationStartCallback (Lcoil3/request/ImageRequest;)Lkotlin/jvm/functions/Function0;
 	public static final fun getAnimationStartCallback (Lcoil3/request/Options;)Lkotlin/jvm/functions/Function0;
+	public static final fun onAnimationEnd (Lcoil3/request/ImageRequest$Builder;Lkotlin/jvm/functions/Function0;)Lcoil3/request/ImageRequest$Builder;
+	public static final fun onAnimationStart (Lcoil3/request/ImageRequest$Builder;Lkotlin/jvm/functions/Function0;)Lcoil3/request/ImageRequest$Builder;
+}
+
+public final class coil3/gif/ImageRequests_androidKt {
 	public static final fun getRepeatCount (Lcoil3/Extras$Key$Companion;)Lcoil3/Extras$Key;
 	public static final fun getRepeatCount (Lcoil3/request/ImageRequest;)I
 	public static final fun getRepeatCount (Lcoil3/request/Options;)I
-	public static final fun onAnimationEnd (Lcoil3/request/ImageRequest$Builder;Lkotlin/jvm/functions/Function0;)Lcoil3/request/ImageRequest$Builder;
```

Comment:
@outadoc Removing these functions breaks binary compatibility. We'll have to keep them and mark them as hidden similar to [here](https://github.com/coil-kt/coil/pull/2695/files#diff-764de0990bc7db62f0f321e7eb262bd0e40db6bb58831a5eb5d4ab2e9016fb26R50).